core/src/main/java/org/elasticsearch/script/ScriptService.java

```diff
@@ -527,6 +528,10 @@ public void onFileInit(Path file) {
             if (logger.isTraceEnabled()) {
                 logger.trace("Loading script file : [{}]", file);
             }
+            if (FileSystemUtils.isHidden(file)) {
+                logger.warn("--- Hidden file skipped : [{}]", file);
```

Comment:
I agree with @jasontedor .  I think `debug` is fine here. If I had to guess this is a leftover from debugging the implementation. I know I tend to use stuff like `---` or `asdfewfca` to mark any temporary logs I add.